.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
Since `linter`, `prettier` and `unit tests` is super small but all depends on `npm install`, I wonder if we should just merge them into one, and `playwright-tests` on it's own workflow